src/dawn/CodeGen/CodeGen.h

```diff
@@ -38,7 +39,11 @@ extern std::map<Key, Value> orderMap(const std::unordered_map<Key, Value>& umap)
 /// @ingroup codegen
 class CodeGen {
 protected:
-  OptimizerContext* context_;
+  std::map<std::string, std::shared_ptr<iir::StencilInstantiation>>& context;
```

Comment:
maybe create an alias

Reply:
since it's only used once afterwards I don't see much benefit. Would it help readability?

Reply:
It's used in each backend.